src/components/shared/Select/README.md

```diff
@@ -12,7 +12,7 @@
 |                       | necessary | types                                                                | default |
 | --------------------- | --------- | -------------------------------------------------------------------- | ------- |
 | testID                |           | string                                                               |         |
-| label                 |           | string                                                               |         |
+| title                 |           | string                                                               |         |
```

Comment:
please add `titleTextStyle` prop

Reply:
Sorry for my sooo late reply!
I've updated my commit to do what you asked.
And thanks a lot for your suggestion :)